iothub/device/src/Common/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -11,17 +11,29 @@
 
 namespace Microsoft.Azure.Devices.Client
 {
+    /// <summary>
+    /// A helper class to build Shared Access Signature tokens
+    /// </summary>
     public class SharedAccessSignatureBuilder
     {
         private string _key;
 
+        /// <summary>
+        /// Initializes a new instance of <see cref="SharedAccessSignatureBuilder"/> class
+        /// </summary>
         public SharedAccessSignatureBuilder()
         {
             TimeToLive = TimeSpan.FromMinutes(60);
         }
 
+        /// <summary>
+        /// Name of the authorization rule
```

Comment:
Since we call this a "shared access policy" in azure terms, how about we call this something like - "The shared access policy name"?